server/api/main.js

```diff
@@ -0,0 +1,36 @@
+import { readFile } from 'fs'
+import { URL } from 'node:url'
+
+const CACHE_TIMEOUT = 3 * 60 * 1000
+
+let responseCache = null
+
+setInterval(() => {
```

Comment:
We don’t need to clean cache (in production mode). We will have file changes only on new deployment (which will stop old server and run new one).

Reply:
Yes, I'm glad that it can be removed)